src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.VirtualRouterTests/TestVirtualRouterPeerCRUD.json

```diff
@@ -4478,7 +4478,7 @@
     }
   ],
   "Names": {
-    "Test-VirtualRouterPeerCRUD": [
+    "Test-HubBgpConnectionCRUD": [
```

Comment:
It seems you modified the test case name in recordings but not source code, so the case is failing

> [xUnit.net 00:08:19.35]     Commands.Network.Test.ScenarioTests.VirtualRouterTests.TestVirtualRouterPeerCRUD [FAIL]

Reply:
resolved